src/OrbitGl/Track.cpp

```diff
@@ -201,8 +108,9 @@ void Track::DoDraw(Batcher& batcher, TextRenderer& text_renderer, const DrawCont
 void Track::DoUpdateLayout() {
   CaptureViewElement::DoUpdateLayout();
 
+  header_->SetWidth(layout_->GetTrackTabWidth());
```

Comment:
Because the track's width may have changed.

Reply:
The method is constant, but the value it returns may still have changed from last frame, the track can't know that. For example, in dev mode, it can be adjusted through a slider in the debug tab.